aziotctl/aziotctl-common/src/check_last_modified.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use std::fs;
+use std::path::Path;
+
+pub enum CheckResult {
+    Ok,
+    Ignored,
+    Warning(String),
+    Failed(std::io::Error),
+}
+
+pub fn check_last_modified(services: &[&str]) -> CheckResult {
+    let config = Path::new("/etc/aziot/config.toml");
+
+    if !config.exists() {
+        return CheckResult::Ignored;
+    }
+
+    let config_metadata = match fs::metadata(config) {
+        Ok(m) => m,
+        Err(err) => return CheckResult::Failed(err),
+    };
+
+    let config_last_modified = config_metadata
+        .modified()
+        .expect("file metadata should contain valid last_modified");
+
+    for service in services {
+        let service_config = format!("/etc/aziot/{}/config.d/00-super.toml", service);
+        let service_config = Path::new(&service_config);
+
+        if !service_config.exists() {
+            return CheckResult::Warning(format!(
+                "{} does not exist.\n\
+                Did you run 'iotedge config apply'?",
```

Comment:
It's not `iotedge` when you run it from `aziotctl`. Use `aziotctl_common::program_name()`

Same for the other `Warning`

Reply:
Thanks, fixed.